### ISSUE TITLE
possible bug in installation script

### ISSUE THREAD

**Issue by ARW-UBT** [jjmiao1314/16sPIP#2]
Hello,
I have installed 16sPIP as described in the help instruction on the www pages. For some reasons, there seems to be in typo in the installation script. Two folders seem not receive the folder name as expected: These are

/bin/6sPIP_BIN
/bin/6sPIP_PATH
Is this as typo (I would expect 16sPIP... instead of 6sPIP...)

The example scripts give in the help menu do not complete sucessufully, the reason could be this type described above. 
Do I have to adjust the REF_PATH on line 20 in 16sPIP.sh?
I have attached the content of /bin and a log file of your example: 

Thanks for your suppor
[log2.txt](https://github.com/jjmiao1314/16sPIP/files/7203497/log2.txt)
[log1.txt](https://github.com/jjmiao1314/16sPIP/files/7203499/log1.txt)
t!

**Comment by Mr-Wornock** [comment]
Have you solved the problem yet?

**Comment by ARW-UBT** [comment]
Yes, I have resolved the issue by manually correcting the bugs in the installation script. 
Best regards

**Comment by Mr-Wornock** [comment]
Could you tell me your solution? I have met the same problem and haven't solved it yet，thank you❤️



发自 网易邮箱大师




---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2021年11月10日 18:33 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.******@***.***> |
| 主题 | Re: [jjmiao1314/16sPIP] possible bug in installation script (#2) |

Yes, I have resolved the issue by manually correcting the bugs in the installation script.
Best regards

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by ARW-UBT** [comment]
I do not have tracks of the steps. But what I did was to look at the error messages produced by the installer.sh scripts and try to get this issue resolved by adaptung to the local machine. One important step was to manually copy any of the produced *.sh scripts also to the /bin/ folder (this is one bug in the script). 
I am not sure whether it made any difference, but I used a fresh virtual installation of Ubuntu 16.04 as 16sPIP has been developed on this version. 
I was not successful on CentOS7 and Ubuntu 18.04 

Best regards
ALfns

**Comment by Mr-Wornock** [comment]
Thank you for your kind attention personal response.


Best regards

**Comment by ericbrzoska** [comment]
Does anyone found an alternative tool?

**Comment by ARW-UBT** [comment]
Hi @ericbrzoska 
have a look at FAPROTAX, it has some pathogenic bacteria in its database.
